src/mtool/api/rest/app.py

```diff
@@ -2063,8 +2063,7 @@ def set_telemetry_config():
         return make_response('Could not configure Telemetry URL'+str(e), 500)
 
 @app.route('/api/v1/checktelemetry', methods=['GET'])
-@token_required
-def check_telemetry(current_user):
+def check_telemetry():
```

Comment:
It is better to add token in the UI, than removing the user check in API

Reply:
Thanks, I have updated.